pkg/apis/storage/validation/validation_test.go

```diff
@@ -2063,12 +2069,6 @@ func TestCSIDriverValidationUpdate(t *testing.T) {
 				new.Spec.FSGroupPolicy = &fileFSGroupPolicy
 			},
 		},
```

Comment:
add a test for changing storage capacity to an invalid value and make sure it fails (it looks like `nil` with CSIStorageCapacity enabled is invalid, even though we don't expect to ever encounter that because the field is defaulted to false if the feature is enabled)

Reply:
Added. To make it work, I had to add a call to `validateCSIDriverSpec` in `ValidateCSIDriverUpdate`. That wasn't necessary before when all fields were immutable but now is good to have, albeit a bit redundant because there is not much that is going to be caught by it. As you said, nil should not occur in practice.